packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js

```diff
@@ -445,6 +446,12 @@ test('amm doubleSwap', async t => {
 
   const ammInstance = await amm.instance;
 
+  const metricsSub = await E(amm.ammPublicFacet).getMetrics();
+  const metricsNotifier = makeNotifierFromAsyncIterable(metricsSub);
+
+  const state0 = await E(metricsNotifier).getUpdateSince();
+  t.deepEqual(state0.value, { XYK: [] });
```

Comment:
the test will be easier to follow using this helper,
```suggestion
  const m = await subscriptionTracker(t, metricsSub);
  m.assertInitial({ XYK: [] });
```

Reply:
done